runtime/src/accounts_db.rs

```diff
@@ -40,7 +40,7 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::{Arc, RwLock};
 use sys_info;
 
-const ACCOUNT_DATA_FILE_SIZE: u64 = 16 * 1024 * 1024;
+const ACCOUNT_DATA_FILE_SIZE: u64 = 4 * 1024 * 1024;
```

Comment:
smaller and smaller ;)

@jackcmay does this look like something to be concerned about?

Reply:
Maybe I can make a special path to create a new larger appendvec if the size is small.

Reply:
added this